.github/workflows/macos11.yml

```diff
@@ -0,0 +1,46 @@
+run-name: macOS-11_${{ vars.current_date }}_unstable.${{ github.run_id }}.${{ github.run_attempt }}
+on:
+  workflow_dispatch:
+    inputs:
+      CUSTOM_REPOSITORY:
+        description: 'Custom repository (owner/repo)'
+        required: false
+      CUSTOM_REPOSITORY_HASH:
```

Comment:
nit: may be `CUSTOM_REPOSITORY_REF` or `CUSTOM_REPOSITORY_COMMIT`?

Reply:
Changed to CUSTOM_REPOSITORY_COMMIT_HASH